tests/Brazil/CarnavalTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ *  This file is part of the Yasumi package.
+ *
+ *  Copyright (c) 2015 - 2016 AzuyaLabs
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ *  @author Dorian Neto <doriansampaioneto@gmail.com>
+ */
+
+namespace Yasumi\tests\Brazil;
+
+use DateTime;
+use DateInterval;
+use Yasumi\Provider\ChristianHolidays;
+
+/**
+ * Class for testing Carnaval in the Brazil.
+ */
+class CarnavalTest extends BrazilBaseTestCase
```

Comment:
All holiday test classes need to implement the contract `YasumiTestCaseInterface`. This to ensure some necessary test methods are included. Please add these for all the holiday test classes.